src/User/Serializers/UserNormalizer.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\User\Serializers;
+
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use VSV\GVQ_API\User\Models\User;
+
+class UserNormalizer implements NormalizerInterface
+{
+    /**
+     * @inheritdoc
+     * @param User $user
+     */
+    public function normalize($user, $format = null, array $context = array())
+    {
+        return [
+            'id' => $user->getId()->toString(),
+            'email' => $user->getEmail()->toNative(),
+            'password' => $user->getPassword()->toNative(),
```

Comment:
Ik denk dat we best `password` nergens vrijgeven. Hiervoor passen we ook best de default constructor van `User` aan en halen `Password` eruit en gebruiken de `withPassword` constructie waarbij we eerst een clone doen en dan het paswoord zetten. Dan kunnen we in de model laag waar nodig al dan niet het paswoord zetten.